### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by mikeller**

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> FAIL
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by blckmn — APPROVED** [review]

**Comment by McGiverGim** [comment]
@blckmn this PR is done automatically by Crowdin and we don't use it. So this must NOT be merged. It contains languages that we don't use, a lot of commits, etc

**Pull request closed without merging**
